lib/runner/cli/cli.js

```diff
@@ -17,6 +17,10 @@ class CliRunner {
     return './nightwatch.conf.cjs';
   }
 
+  static get CONFIG_FILE_TS() {
+    return './nightwatch.conf.ts';
```

Comment:
@vaibhavsingh97 we need a test that checks for this.

Reply:
Added